examples/chicago_taxi/preprocess.py

```diff
@@ -95,7 +95,7 @@ def preprocessing_fn(inputs):
 
     for key in taxi.VOCAB_FEATURE_KEYS:
       # Build a vocabulary for this feature.
-      outputs[taxi.transformed_name(key)] = transform.string_to_int(
+      outputs[taxi.transformed_name(key)] = transform.compute_and_apply_vocabulary(
```

Comment:
Can you format the lines like so, so that each line is less than 80 characters long?
```
    for key in taxi.VOCAB_FEATURE_KEYS:
      # Build a vocabulary for this feature.
      outputs[
          taxi.transformed_name(key)] = transform.compute_and_apply_vocabulary(
              _fill_in_missing(inputs[key]),
              top_k=taxi.VOCAB_SIZE,
              num_oov_buckets=taxi.OOV_SIZE)
```

Reply:
No problem. Fixed!